### PR TITLE
[AIRFLOW-5828] Move build logic out from hooks/build

### PR DESCRIPTION
**Pull request by potiuk**
This is the final step of simplifying the Breeze scripts by
moving all the logic out from Travis' hooks/build

---
Issue link: [AIRFLOW-5828](https://issues.apache.org/jira/browse/AIRFLOW-5828)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Hey @dimberman @nuclearpinguin @mik-laj @kaxil -> this is the final stage of simplifying the build scripts from breeze - now all the build logic is nicely separated out into functions and travis's hooks/build has no logic -> everything is in the scripts/ci/_utils.sh as functions. This is final stage of #6500 which I am closing now.

File: breeze [hunk 1]
```diff
@@ -128,15 +125,9 @@ function setup_default_breeze_variables() {
     export INSTALL_AIRFLOW_VERSION=${INSTALL_AIRFLOW_VERSION:="current"}
 
     # Determine version of the Airflow from version.py
-    AIRFLOW_VERSION=$(cat airflow/version.py - << EOF | python
-print(version.replace("+",""))
-EOF
-    )
+    AIRFLOW_VERSION=$(grep version <"airflow/version.py" | awk '{print $3}' | sed "s/['+]//g")
```
Comment (mik-laj) [inline breeze]:
```suggestion
    AIRFLOW_VERSION=$(grep version "airflow/version.py" | awk '{print $3}' | sed "s/['+]//g")
```

Comment (potiuk) [reply breeze]: Done.

**Comment by potiuk** [comment]
@nuclearpinguin @mik-laj -> this one makes the build scripts much, much better. PTAL. I'd love to finally merge that one and complete all the vastly simplify of breeze scripts :)

**Review by turbaszek — APPROVED** [review]

**Comment by nuclearpinguin** [comment]
LGTM :)

**Pull request merged**